usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -62,14 +63,24 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
         )
     )
 
+    scap_report = os.environ.get("SCAP_REPORT", "")
+    if scap_report:
+        scap_report = "--report {}".format(scap_report)
```

Comment:
We probably shouldn't blindly take any user set value. For example the test running as root with an environment variable setting of `/boot/initrd` would render the system unbootable. We need to write the file to a set location that we know not to be dangerous.

Reply:
> We probably shouldn't blindly take any user set value. For example the test running as root with an environment variable setting of `/boot/initrd` would render the system unbootable. We need to write the file to a set location that we know not to be dangerous.

Good point. I will add a check that the file must not exist and the filename must end with ".html".

Reply:
> We probably shouldn't blindly take any user set value. For example the test running as root with an environment variable setting of `/boot/initrd` would render the system unbootable. We need to write the file to a set location that we know not to be dangerous.

Fixed.